src/internal/packager/create.go

```diff
@@ -114,10 +114,17 @@ func addComponent(tempPath tempPaths, component types.ZarfComponent) {
 		re := regexp.MustCompile(`\.git$`)
 		for _, chart := range component.Charts {
 			isGitURL := re.MatchString(chart.Url)
+			URLLen := len(chart.Url)
 			if isGitURL {
 				_ = helm.DownloadChartFromGit(chart, componentPath.charts)
-			} else {
+			} else if URLLen > 0 {
```

Comment:
We'll likely be arguing pedantics over which takes priority (`URLLen`  else using the `LocalPath`) - but is there anything that prevents a user from using both and/or documentation that establishes precedence?

Reply:
Input is now validated to only have one of url, localpath or gitpath